docs/sphinx_docs/Input_file_formats/Native/02_types/02_01_builtin.md

```diff
@@ -129,7 +127,73 @@ goal g:
     x + t = 0
 ```
 
-### Bitvectors
+## Additional numeric primitives
+
+Alt-Ergo provides built-in primitives for mixed integers and real problems,
+along with some limited reasoning support, typically restricted to constants.
+These primitives are only available in the native input format.
+
+### Conversion between integers and reals
+
+```alt-ergo
+logic real_of_int : int -> real
+```
+
+`real_of_int` converts an integer into its representation as a real number.
+
+*Note*: When using the `dolmen` frontend, `real_of_int` is also available in
+the smtlib2 format as the `to_real` function from the `Reals_Ints` theory.
+
+```alt-ergo
+logic int_floor : real -> int
+logic int_ceil : real -> int
+logic real_is_int : real -> bool
+```
+
+`int_floor` and `int_ceil` implement the standard `floor` and `ceil` functions.
+They compute the greatest integer less than a real and the least integer
+greater than a real, respectively.
```

Comment:
By standard, you mean the SMT-LIB standard or the mathematical one? I guess they agree but who knows :)

Reply:
There are no `floor` or `ceil` functions in the SMT-LIB standard. Changed to usual instead.